content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -290,13 +290,16 @@ parameters:
   type: io1
   iopsPerGB: "50"
   encrypted: "true"
+  tagSpecification: "key1=value1"
 allowedTopologies:
 - matchLabelExpressions:
   - key: topology.ebs.csi.aws.com/zone
     values:
     - us-east-2c
 ```
 
+- `tagSpecification`: Tags to be applied on dynamically provisioned ebs volumes through AWS_EBS_CSI_DRIVER (in v1.6.0 and later).
```

Comment:
I think it's better to link to the [AWS_EBS_CSI_DRIVER StorageClass Tagging]( https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/tagging.md) so that users can refer to it.